kernel/kernel-defaults/src/test/scala/io/delta/kernel/defaults/utils/TestUtils.scala

```diff
@@ -575,4 +581,11 @@ trait TestUtils extends Assertions with SQLHelper {
         })
     }
   }
+
+  /**
+   * Returns a URI encoded path of the resource.
+   */
+  def getTestResourceFilePath(resourcePath: String): String = {
+    ResourceLoader.classLoader.getResource(resourcePath).getFile
```

Comment:
Null check `getResource` return value and throw proper error ("resource not found")